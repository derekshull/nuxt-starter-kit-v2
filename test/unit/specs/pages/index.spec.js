import { shallowMount } from "@vue/test-utils";
import Page from "~/pages/index.vue";

describe("Page /", () => {
  it("renders to a snapshot", () => {
    const wrapper = shallowMount(Page);
    expect(wrapper.element).toMatchSnapshot();
  });
});

// const localVue = createLocalVue();

// localVue.use(Vuex);

// describe('Page homepage', () => {
//   let store;
//   const mockLogo = 'mock logo';

//   beforeEach(() => {
//     store = new Vuex.Store({
//       actions: {
//         'logo/logoAction': jest.fn(),
//       },
//       getters: {
//         'logo/logo': () => mockLogo,
//       },
//     });
//   });

//   it('should mock store correctly', () => {
//     const wrapper = shallow(Page, { store, localVue });
//     expect(wrapper.vm.logo).toEqual(mockLogo);
//   });
// });
