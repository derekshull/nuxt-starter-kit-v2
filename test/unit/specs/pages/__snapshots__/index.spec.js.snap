// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page / renders to a snapshot 1`] = `
<section
  class="container"
>
  <form>
    <div
      style="margin-top: 10px;"
    >
      <label
        for="username"
      >
        Username
      </label>
       
      <input
        autofocus="autofocus"
        name="username"
        type="text"
      />
    </div>
     
    <div
      style="margin-top: 10px;"
    >
      <label
        for="password"
      >
        Password
      </label>
       
      <input
        name="password"
        type="password"
      />
    </div>
     
    <div
      style="margin-top: 10px;"
    >
      <input
        type="submit"
        value="Log In"
      />
    </div>
  </form>
</section>
`;
